.github/workflows/test-packaging.yml

```diff
@@ -62,8 +62,14 @@ jobs:
     - name: Build DEB
       run: ./build/fleetctl package --type deb --enroll-secret=foo --fleet-url=https://localhost:8080
 
+    - name: Build DEB with Fleet Desktop
+      run: ./build/fleetctl package --type deb --enroll-secret=foo --fleet-url=https://localhost:8080 -fleet-desktop
```

Comment:
Should the last flag be `--fleet-desktop`?

Reply:
I had one job!

Thanks for catching it!

Reply:
I didn't realize either 😅